apps/epl_ets/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: prop propclean
+
+prop:
+	rebar3 as test proper
+
+propclean:
```

Comment:
Hmm `propclean` target suggest that proper-specific stuff will be cleaned - not the whole build.